scripts/orca-migrate-container

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# ======================= Load Balance ======
+
+#Check if user is root
+if [[ $EUID -ne 0 ]]; then
+        echo $EUID
+        echo "Script must be run as root" 1>&2
+        exit 1
+fi
```

Comment:
Let's see if we can get the script to work without root. What does it need root for?

Reply:
Oh I wasn't sure if there was a way to make it so that the user doesn't have to enter their password multiple times throughout this process. If they never had it saved. (uptime, scp to other server)